### DISCUSSION TITLE
Greek Language Leader

### DISCUSSION THREAD

**Discussion by chatziko** [OHF-Voice/intents#2467] [Language Leaders]
I'd like to apply for greek language leader. I'm involved in HA for a long time, made a few PRs in this repo and lots of other contributions in HA-related projects (both voice related and not).

**Comment by synesthesiam** [comment]
Thanks for volunteering! I've also appreciated your work on Wyoming satellites :slightly_smiling_face: 
I'll get you added as a language leader :+1:

**Comment by synesthesiam** [comment]
https://github.com/home-assistant/intents/pull/2709

**Reply by chatziko** [reply]
Awesome, thanks, hopefully I can find some time soon to move the state of the greek language forward.
